docs/faq.md

```diff
@@ -80,3 +81,28 @@ Currently, the main benefit of [signed releases](/checks.md#signed-releases) is
+
+It's important to reiterate that Branch-Protection score is Tier-based. If a setting from Tier 1 is not satisfied, it does not matter that all other settings are met, the score will be truncated up the Tier's maximum. In this case, 3/10. The following table shows the relation between branch protection settings on GitHub and the score Tier:
+
+| Name                                                                                                     | Status                          | Required only for admin token | Tier |
+| -------------------------------------------------------------------------------------------------------- | ------------------------------- | ----------------------------- | ---- |
+| Allow force pushes                                                                                       | Disabled                        | -                             | 1    |
+| Allow deletions                                                                                          | Disabled                        | -                             | 1    |
+| Do not allow bypassing the above settings                                                                | Enabled                         | Yes                           | 1    |
+| Require a pull request before merging > Require Approvals                                                | Enabled with at least 1         | -                             | 2    |
+| Require status checks to pass before merging > Require branches to be up to date before merging          | Enabled                         | Yes                           | 2    |
+| Require a pull request before merging > Require approval of the most recent reviewable push              | Enabled                         | Yes                           | 2    |
+| Require status checks to pass before merging > Status Checks                                             | At least 1                      | -                             | 3    |
+| Require a pull request before merging > Require Approvals                                                | Enabled with at least 2         | -                             | 4    |
+| Require a pull request before merging > Require review from Code Owners                                  | Enabled and has CODEOWNERS file | -                           | 4    |
+| Require a pull request before merging > Dismiss stale pull request approvals when new commits are pushed | Enabled                         | Yes                           | 5    |
```

Comment:
these tiers will need to be changed to reflect the actual tiers (see comments in checks.md)

Reply:
Check explanations above.